src/templates/_common/components/deriv_banner.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const DerivBanner = () => (
+const DerivBanner = ({ dynamic_link }) => (
```

Comment:
can we use a better name such as content_name etc?

Reply:
Sure :+1: